subiquity/server/server.py

```diff
@@ -313,9 +314,11 @@ def __init__(self, opts, block_log_dir):
                     "examples", "snaps"),
                 self.scale_factor, opts.output_base)
             self.snapd = AsyncSnapd(connection)
+            self.snapdapi = make_api_client(self.snapd)
         elif os.path.exists(self.snapd_socket_path):
             connection = SnapdConnection(self.root, self.snapd_socket_path)
             self.snapd = AsyncSnapd(connection)
+            self.snapdapi = make_api_client(self.snapd)
         else:
             log.info("no snapd socket found. Snap support is disabled")
```

Comment:
For the current support this is appropriate but that might need to be upgraded to a more serious error later on.

Reply:
this is here because the wsl system setup code comes through here (we need to fix this somehow or other!)